DevAcademy/Scenes/Places/PlacesScene.swift

```diff
@@ -44,23 +36,11 @@ struct PlacesScene: View {
         .navigationBarTitleDisplayMode(.inline)
     }
 
-    func tapped(on feature: Feature) {
-    }
-
-    func fetch() {
-        DataService.shared.fetchData { result in
-            switch result {
-            case .success(let features):
-                self.features = features.features
-            case .failure(let error):
-                print(error)
-            }
-        }
-    }
+   
 }
 
-struct PlacesScene_Previews: PreviewProvider {
-    static var previews: some View {
-        PlacesScene()
-    }
-}
+//struct PlacesScene_Previews: PreviewProvider {
+//    static var previews: some View {
+//        PlacesScene()
```

Comment:
Tady kdybys to chtel upravit a odkomentovat, tak jenom nezapomenout pridat ty `.environmentObjecty`, stejne jako se pridavaji v `DevAcademyApp` aby ti to preview necrashovalo.